src/site/content/en/lighthouse-performance/lighthouse-max-potential-fid/index.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: Max Potential First Input Delay
+description: |
+  Learn about Lighthouse's Max Potential First Input Delay metric and
+  how to measure and optimize it.
+date: 2019-05-02
+updated: 2019-10-16
+web_lighthouse:
+  - max-potential-fid
+---
+
+Max Potential First Input Delay (FID) is one of the metrics
+tracked in the **Performance** section of the Lighthouse report.
+Each metric captures some aspect of page load speed.
+
+Lighthouse displays Max Potential FID in milliseconds:
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="max-potential-fid.png" alt="A screenshot of the Lighthouse Max Potential First Input Delay audit">
+</figure>
+
+## What Max Potential FID measures
+
+Max Potential FID measures the duration of the
+[longest task](/long-tasks-devtools#what-are-long-tasks) after
+[First Contentful Paint][fcp]. It is meant to capture the worst case
```

Comment:
this looks great to me, I might want to throw in just one more sentence explaining that this means the delay between when your users try to interact with the page and it can actually start responding so they don't have to click to figure that out, but I'll leave to your discretion on how you'd prefer to overlap/not overlap the docs

Reply:
Good call, I added a sentence explaining what FID is